CHANGELOG.md

```diff
@@ -2,6 +2,13 @@
 
 ## Release 126.0.2 (in development)
 
+### Bug fixes
+
+- Force bootstrap roles on ext_pillar to prevent bootstrap
+  deployment to fail when re-run
+  (see ISSUE[2137](https://github.com/scality/metalk8s/issues/2137)).
+  (PR[4196](https://github.com/scality/metalk8s/pull/4196))
+
```

Comment:
Let's have the same syntax as before and let's also reword it a bit 

```suggestion
- [2137](https://github.com/scality/metalk8s/issues/2137) - Fix a bug
  that prevents re-run of the bootstrap script if it fails at a specific point
  (PR[4196](https://github.com/scality/metalk8s/pull/4196))

```

Reply:
Done !!!